packages/cms/src/schemas/objects/milestone.ts

```diff
@@ -0,0 +1,30 @@
+export default {
+  title: 'Single milestone',
+  name: 'milestone',
+  type: 'object',
+  fields: [
+    {
+      title: 'Datum',
+      name: 'date',
+      type: 'date',
+      options: {
+        dateFormat: 'YYYY-MM-DD',
+      },
+      validation: (Rule: any) => Rule.required(),
+    },
+    { name: 'title', type: 'localeString', title: 'Titel' },
+  ],
+  preview: {
+    select: {
+      title: 'title.nl',
+      date: 'date'
+    },
+    prepare(selection: any) {
+      const {title, date} = selection
```

Comment:
Do you happen to know the types of `title` and `date`? If so, can you please type them?
```suggestion
    prepare(selection: { title: string; date: ??? }) {
      const {title, date} = selection
```